issuers/vault/vault_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"crypto/rand"
 	"crypto/tls"
 	"crypto/x509"
+	"github.com/spiffe/go-spiffe/uri"
```

Comment:
TIL about this library. Could write something like this to validate the otherSANs as well I guess. Could you move this import to the other third party imports? Also I think you need to run `go mod tidy` in this repo to update the `go.mod` file with this new dependency.

Reply:
I realised the go mod tidy mistake just after I pushed. Fixed that and reordered the imports.

It's probably similar logic to validate the otherSANs as is used in the go-spiffe library I imagine. I had a quick look at it though and there's some non-trivial magic values around the place though (maybe it's simpler for otherSANs if those OIDs are the same as provided to Vault?).

Reply:
I'll raise an issue about adding better tests for otherSANs now that I've got the spiffe library to base it on, thanks!